src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/AddSmallProducerSubmissionHandler.cs

```diff
@@ -54,21 +54,28 @@ public async Task<Guid> HandleAsync(AddSmallProducerSubmission request)
 
             string producerRegistrationNumber;
 
-            if (existingProducer != null)
+            if (!string.IsNullOrWhiteSpace(directRegistrant.ProducerRegistrationNumber))
             {
-                producerRegistrationNumber = existingProducer.ProducerRegistrationNumber;
+                producerRegistrationNumber = directRegistrant.ProducerRegistrationNumber;
```

Comment:
@will-hope98  should create new integration test in AddSmallProducerSubmissionHandlerTests that tests this.

Update the DirectRegistrantDbSetup class to have a WithPrn(..
Something like
![image](https://github.com/user-attachments/assets/0fa5c142-3a8f-4531-b2d1-ab06c39b7893)